src/lib/cache/memory-cache.ts

```
@@ -0,0 +1,242 @@
+      max: options.max || 500, // Maximum 500 items
+      ttl: options.ttl || 1000 * 60 * 5, // 5 minutes default TTL
+      updateAgeOnGet: true,
+      updateAgeOnHas: true,
+    });
+    
+    // Check if we should persist to database
+    this.persistToDB = process.env.CACHE_PERSIST === 'true';
+  }
+
+  static getInstance(): MemoryCache {
+    if (!this.instance) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)